Example/ExampleApp/ViewModel/PaymentHandler.swift

```diff
@@ -19,24 +19,20 @@ class PaymentHandler: NSObject {
     func startPayment(completion: @escaping PaymentCompletionHandler) {
         completionHandler = completion
         
-        // Create our payment request
         let paymentRequest = PKPaymentRequest()
         paymentRequest.paymentSummaryItems = [.init(label: "Test Company recieving the payment", amount: .one, type: .final)]
         paymentRequest.merchantIdentifier = applePayMerchantId
         paymentRequest.countryCode = "HU"
         paymentRequest.currencyCode = CurrencyCodes.HUF.rawValue
         paymentRequest.merchantCapabilities = [.capability3DS]
         paymentRequest.supportedNetworks = [.visa, .masterCard, .maestro]
-        // paymentRequest.requiredShippingContactFields = [.phoneNumber, .emailAddress]
-
-        // Display our payment request
+        
         paymentController = PKPaymentAuthorizationController(paymentRequest: paymentRequest)
         paymentController?.delegate = self
         paymentController?.present(completion: { (presented: Bool) in
             if presented {
```

Comment:
the if statement can be flipped: `if !presented` to save on a few lines

Reply:
True! This apple pay implementation in the example is temporary. As soon as I get it to work properly, I want to refactor and integrate it into the library.